backend/app/db/models/document.py

```diff
@@ -288,3 +288,29 @@ class Event(Base):  # noqa: D101
     name = sa.Column(sa.Text, nullable=False)
     description = sa.Column(sa.Text, nullable=False)
     created_ts = sa.Column(sa.DateTime(timezone=True), nullable=False)
+
+
+class Relationship(Base):  # noqa: D101
+    __tablename__ = "relationship"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    type = sa.Column(sa.Text, nullable=False)
+    name = sa.Column(sa.Text, nullable=False)
+    description = sa.Column(sa.Text, nullable=False)
+
+
+class DocumentRelationship(Base):  # noqa: D101
+    __tablename__ = "document_relationship"
+
+    document_id = sa.Column(
+        sa.Integer,
+        sa.ForeignKey(Document.id, ondelete="CASCADE"),
```

Comment:
can I just check that this doesn't mean that deleting a relationship will delete the associated document?

Reply:
I believe so -as I checked the document - I may have misunderstood so its worth me writing a test for this :+1:

Reply:
There's now a test for this